plugin/sampling/strategystore/static/constants.go

```diff
@@ -35,7 +35,7 @@ const (
 var (
 	// defaultStrategy is the default sampling strategy the Strategy Store will return
 	// if none is provided.
-	defaultStrategy = sampling.SamplingStrategyResponse{
+	defaultStrategyResponse = sampling.SamplingStrategyResponse{
```

Comment:
Would it be better to make this into a function? At one point we're doing deepCopy(), instead we can just always return a new object.